lib/chat_web/live/main_live/admin/cargo_camera_sensors_form.ex

```diff
@@ -2,21 +2,21 @@ defmodule ChatWeb.MainLive.Admin.CargoCameraSensorsForm do
   @moduledoc """
   Handles showing and updating camera sensors settings form data.
   """
-
   use ChatWeb, :live_component
 
   import Phoenix.Component
 
   alias Chat.Admin.CargoSettings
   alias Chat.AdminRoom
+  alias Chat.Sync.Camera.Sensor
 
   def mount(socket) do
     cargo_settings = AdminRoom.get_cargo_settings()
 
     socket
     |> assign(:cargo_settings, cargo_settings)
     |> assign(:changeset, cargo_settings |> CargoSettings.camera_sensors_changeset())
-    |> assign(:invalid_sensors, [])
```

Comment:
The reason that `:invalid_sensors` was presented as a list is that there are no static indexes in the list during the delete urls in the form. It does avoid the scenario when i have 3 invalid urls added in the form to validate for example. I guess it's needed to have some re-indexation mechanism for the `:invalid_sensors` state during the `validate` and `delete`.